packages/terra-action-footer/package.json

```diff
@@ -23,7 +23,8 @@
   "homepage": "https://github.com/cerner/terra-core#readme",
   "peerDependencies": {
     "react": "^16.8.5",
-    "react-dom": "^16.8.5"
+    "react-dom": "^16.8.5",
+    "terra-theme-context": "^1.0.0"
```

Comment:
Shouldn't this be a dependency?

Reply:
Updated here https://github.com/cerner/terra-core/pull/2955/commits/6146fb9e5371674f76085d02d1f01acd52f3d3b4